From 129ad44fa8dc4e3cfb0dcc82df1421861ba6f3eb Mon Sep 17 00:00:00 2001
From: Colin Guthrie <colin@mageia.org>
Date: Fri, 6 Apr 2012 11:21:27 +0100
Subject: [PATCH 509/509] mageia: Make the display-manager-failure service
 come after shutdown.target

This prevents it kicking in when shutting down or rebooting with some
display managers.
---
 units/mageia/display-manager-failure.service |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/units/mageia/display-manager-failure.service b/units/mageia/display-manager-failure.service
index 291cd46..bd748d1 100644
--- a/units/mageia/display-manager-failure.service
+++ b/units/mageia/display-manager-failure.service
@@ -1,6 +1,6 @@
 [Unit]
 Description=Display Manager Failure Message
-After=dev-tty1.device systemd-user-sessions.service plymouth-quit-wait.service graphical.target
+After=dev-tty1.device systemd-user-sessions.service plymouth-quit-wait.service graphical.target shutdown.target
 
 Conflicts=getty@tty1.service
 After=getty@tty1.service
-- 
1.7.9.3

